WooCommerce/src/main/kotlin/com/woocommerce/android/ui/mystore/MyStoreStatsView.kt

```diff
@@ -189,6 +196,12 @@ class MyStoreStatsView @JvmOverloads constructor(
         isRequestingStats = true
     }
 
+    fun updateCustomDateRange(customDateRange: Pair<Date, Date>?) {
+        customRange = customDateRange
+        customRangeButton.isVisible = customDateRange == null && FeatureFlag.CUSTOM_RANGE_ANALYTICS.isEnabled()
+        customRangeTab.view.isVisible = customDateRange != null && FeatureFlag.CUSTOM_RANGE_ANALYTICS.isEnabled()
+    }
```

Comment:
Np. I'd add here the following, to ensure that the custom range tab get selected and visible when adding the custom range. 
```suggestion
        tabLayout.selectTab(customRangeTab)
        tabLayout.scrollX = tabLayout.width
    }
```

Reply:
Cool, thanks Jorge!